pegasus/sites.v3/code.org/public/alt-about/our-impact.md

```diff
@@ -167,10 +167,12 @@ Whether it’s organizing the worldwide Hour of Code, advocating for policy chan
 
 <hr/>
 
-<a href="https://www.guidestar.org/profile/46-0858543" target="_blank" rel="noopener noreferrer">
-  <object data="https://widgets.guidestar.org/gximage2?o=9218725&l=v4" type="image/png" style="vertical-align: middle; margin-right: 40px">
-    <img src="/images/guidestar-logo.png" />
-  </object>
+<a href="https://www.guidestar.org/profile/46-0858543" target="_blank" style="display: inline-block; position: relative; z-index: 1">
+  <span style="display: inline-block; vertical-align: middle; margin-right: 40px">
+    <object data="https://widgets.guidestar.org/gximage2?o=9218725&l=v4" type="image/svg+xml" style="position: relative; z-index: -1">
+      <img src="/images/guidestar-logo.svg" />
+    </object>
+  </span>
```

Comment:
Could we de-dupe this the content in `amazon.md`?

Reply:
i didn't know how to include a "partial" in a .md file, and since i'm pretty sure both this and amazon.md are obsolete i didn't think it was worth taking extra time to figure it out. let me see if i can verify that it's safe to delete these files instead. let me know if you have any tips on how to do that!

Reply:
Oh, I think a `.md.erb` file can have a `<%= view` style partial.  But good if pages can be deprecated.  For the `alt-about` page, we can probably check Dropbox history on that file to see who created it.  For `public/donate/amazon` we can check Google Analytics for current traffic, and we might want to consider redirecting its traffic if it gets much.